tests/unit/docs/test_doc_summarizer.py

```diff
@@ -24,8 +26,57 @@ def test_is_query_helper_subclass():
 def test_summarize(storage_context, service_context):
     """Basic test for DocsSummarizer using mocked index and query engine."""
     config.init_empty_config()
+    config.ols_config.reference_content = ReferenceContent(None)
+    config.ols_config.reference_content.product_docs_index_path = "./invalid_dir"
+    config.ols_config.reference_content.product_docs_index_id = "product"
     summarizer = DocsSummarizer()
     question = "What's the ultimate question with answer 42?"
     summary, documents = summarizer.summarize("1234", question)
     assert question in summary
     assert len(documents) == 0
+
+
+@patch(
+    "ols.src.query_helpers.docs_summarizer.LLMLoader",
+    new=mock_llm_loader(mock_langchain_interface("test response")()),
+)
+@patch("ols.src.query_helpers.docs_summarizer.ServiceContext.from_defaults")
+@patch("ols.src.query_helpers.docs_summarizer.StorageContext.from_defaults")
+@patch(
+    "ols.src.query_helpers.docs_summarizer.load_index_from_storage", new=MockLlamaIndex
+)
+@patch.dict(os.environ, {"TEI_SERVER_URL": "localhost"}, clear=True)
```

Comment:
this environment variable shouldn't be used by the code any more.  @syedriko did we miss one or has a PR not merged yet?

Reply:
fixed